mvpa2/datasets/sources/native.py

```diff
@@ -0,0 +1,110 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Loaders for PyMVPA's own demo datasets"""
```

Comment:
may be make it "pymvpa_demo.py" instead of "native.py" to be more explicit/less confusing?